composeApp/src/iosMain/kotlin/org/ooni/probe/SetupDependencies.kt

```diff
@@ -79,3 +84,19 @@ private fun readAssetFile(path: String): String {
 private class BundleMarker : NSObject() {
     companion object : NSObjectMeta()
 }
+
+@OptIn(kotlinx.cinterop.ExperimentalForeignApi::class)
```

Comment:
We're using setting our OptIn in the build.gradle file. Do you prefer setting it by file?